src/EA.Weee.DataAccess/Mappings/DirectRegistrantMapping.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace EA.Weee.DataAccess.Mappings
+{
+    using EA.Weee.Domain.Producer;
+    using System.Data.Entity.ModelConfiguration;
+
+    public class DirectRegistrantConfiguration : EntityTypeConfiguration<DirectRegistrant>
+    {
+        public DirectRegistrantConfiguration()
+        {
+            ToTable("DirectRegistrant", "Producer");
+
+            HasKey(e => e.Id);
+
+            Property(e => e.RowVersion).IsRowVersion();
+
+            HasRequired(e => e.Organisation).WithMany().HasForeignKey(e => e.OrganisationId);
+            HasOptional(e => e.SICCode).WithMany().HasForeignKey(e => e.SICCodeId);
```

Comment:
I may not be remembering this correctly, but did we not say during estimation we weren't including the SIC code?

Reply:
It could be making a return :)